packages/jest-cli/src/cli/args.js

```diff
@@ -109,6 +109,13 @@ const options = {
       ' prevent snapshots from being written unless explicitly requested.',
     type: 'boolean',
   },
+  clearCache: {
+    default: undefined,
+    description:
+      'Clear default directory Jest uses for cache and exits.' +
```

Comment:
Nit: Clears or exit, whatever the convention is right now (hard to access on a phone)